ApollosProject/apollos-apps
# chore: Update Share for Gratitude

## ✏️ Description
After some feedback, we've decided to make 2 updates to the Gratitude Share screen in order to help clarify what happens when submitting a Gratitude:

1. Remove the text underneath the Gratitude post
2. Change the "Share" button to "Save"

## 🔬 To Test
1. Load up the Daily tab
2. Launch the Gratitude Experience
3. Write a Gratitude and press "Done"
4. Check out the changes on the Share Sheet

## 📸 Screenshots
| Before | After |
| --- | --- |
| ![Screenshot_20221213-150939](https://user-images.githubusercontent.com/45076058/207438770-d559ba6e-b24e-44cf-aca1-2d382c900e1a.png) | ![Screenshot_20221213-153810](https://user-images.githubusercontent.com/45076058/207438804-f4edc367-7bce-4a5f-8320-e91c43bbe1d3.png) |

## ⚠️ To-do Before Merge
n/a

## 📝 Checklist
